data/darktableconfig.xml.in

```diff
@@ -23,11 +23,14 @@
         <option>on startup</option>
         <option>on close</option>
         <option>on both</option>
+        <option>on startup (don't ask)</option>
+        <option>on close (don't ask)</option>
+        <option>on both (don't ask)</option>
       </enum>
     </type>
-    <default>on close</default>
-    <shortdescription>when to check for database maintenance</shortdescription>
-    <longdescription>this will set when db size is checked for possible maintenance condition and ask user to confirm</longdescription>
+    <default>on startup</default>
```

Comment:
I think we agreed to let this be "on close" for now.

Reply:
Ok... but i still think `on startup` might be a bit better choice. Just needs more input to confirm (or deny).